dmoj/executors/mixins.py

```diff
@@ -11,39 +11,79 @@
 from dmoj.judgeenv import env
 from dmoj.utils import setbufsize_path
 from dmoj.utils.unicode import utf8bytes
+from dmoj.executors.filesystem_policies import Rule, ExactDir, RecursiveDir, ExactFile
```

Comment:
Nit: dmoj imports should be alphabetically sorted. It is unfortunate flake is not enforcing this.

Reply:
https://github.com/tlocke/flake8-alphabetize can we make use of this?

Reply:
Probably https://github.com/PyCQA/flake8-import-order (by the simple metric of having more stars ⇒ more likely to stick around), but yeah. If you want to add this it'd be appreciated, but should be done in a separate PR.